README.md

```diff
@@ -1,138 +1,100 @@
-# social-diversity-for-children
+# Social Diversity for Children Foundation
 
-This project was generated with [superplate](https://github.com/pankod/superplate).
+Program registration platform for [Social Diversity for Children Foundation](https://www.socialdiversity.org).
```

Comment:
I think it would be nice to get a bit more detail here - what exactly is it that we're building? It would also be good to mention it's currently in development.

Reply:
Added a TODO